changelogs/fragments/ec2_key-fix-security-vulnerability.yml

```diff
@@ -0,0 +1,3 @@
+---
+breaking_changes:
```

Comment:
Is this still a breaking change?  I believe it's just a new feature and we can backport to stable-6